test/scripts/jenkins_visual_regression.sh

```diff
@@ -1,10 +1,18 @@
 #!/usr/bin/env bash
 
-source test/scripts/jenkins_test_setup_xpack.sh
+source src/dev/ci_setup/setup_env.sh
 source "$KIBANA_DIR/src/dev/ci_setup/setup_percy.sh"
 
-checks-reporter-with-killswitch "Kibana visual regression tests" \
-  yarn run percy exec -t 500 \
+echo " -> building and extracting OSS Kibana distributable for use in functional tests"
```

Comment:
I'll plan to take a pass over these shell scripts to clean them up at some point, so that this doesn't need to be duplicated